From e246e215c18c5f2911979857fc1c0d6f42ed18a2 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Fri, 12 Sep 2014 18:11:20 -0400
Subject: [powerpc] powernv: Fix typos in sysparam code

Message-id: <1410545655-205645-452-git-send-email-dzickus@redhat.com>
Patchwork-id: 94616
O-Subject: [RHEL7 PATCH 451/626] powerpc/powernv: Fix typos in sysparam code
Bugzilla: 1127366
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: David Gibson <dgibson@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1127366

commit 16003d235bdb4d1ebcecfe4915c6e6d8c95f4a0c
Author: Joel Stanley <joel@jms.id.au>
Date:   Thu Apr 24 16:55:36 2014 +0930

    powerpc/powernv: Fix typos in sysparam code

    Signed-off-by: Benjamin Herrenschmidt <benh@kernel.crashing.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/powerpc/platforms/powernv/opal-sysparam.c b/arch/powerpc/platforms/powernv/opal-sysparam.c
index 83f5f2a..c59e77a 100644
--- a/arch/powerpc/platforms/powernv/opal-sysparam.c
+++ b/arch/powerpc/platforms/powernv/opal-sysparam.c
@@ -222,13 +222,13 @@ void __init opal_sys_param_init(void)
  }
 
  if (of_property_read_u32_array(sysparam, "param-len", size, count)) {
-  pr_err("SYSPARAM: Missing propery param-len in the DT\n");
+  pr_err("SYSPARAM: Missing property param-len in the DT\n");
   goto out_free_perm;
  }
 
 
  if (of_property_read_u8_array(sysparam, "param-perm", perm, count)) {
-  pr_err("SYSPARAM: Missing propery param-perm in the DT\n");
+  pr_err("SYSPARAM: Missing property param-perm in the DT\n");
   goto out_free_perm;
  }
 
-- 
1.7.1